homeassistant/components/subaru/config_flow.py

```diff
@@ -0,0 +1,162 @@
+"""Config flow for Subaru integration."""
+from datetime import datetime
+import logging
+
+from subarulink import (
+    Controller as SubaruAPI,
+    InvalidCredentials,
+    InvalidPIN,
+    SubaruException,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.const import (
+    CONF_DEVICE_ID,
+    CONF_PASSWORD,
+    CONF_PIN,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client, config_validation as cv
+
+from .const import (
+    CONF_HARD_POLL_INTERVAL,
+    DEFAULT_HARD_POLL_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    DOMAIN,
+    MIN_HARD_POLL_INTERVAL,
+    MIN_SCAN_INTERVAL,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_instances(hass):
+    """Return a set of configured Subaru instances."""
+    return {entry.title for entry in hass.config_entries.async_entries(DOMAIN)}
+
+
```

Comment:
We should check for CONF_USERNAME in the entry not the title

Reply:
`self._async_current_entries()` might make more sense

Reply:
I'm now using `self._async_current_entries()`, thanks for the suggestion